packages/react/src/primitives/Expander/ExpanderItem.tsx

```diff
@@ -0,0 +1,65 @@
+import classNames from 'classnames';
+import { Item, Header, Trigger, Content } from '@radix-ui/react-accordion';
+
+import { IconExpandMore } from '../Icon';
+import { View } from '../View';
+import { Primitive } from '../types/view';
+import { ExpanderItemProps } from '../types/expander';
+import { ComponentClassNames } from '../shared/constants';
+import { splitPrimitiveProps } from '../shared/styleUtils';
+import { useStableId } from '../shared/utils';
+
+export const EXPANDER_ITEM = 'expander-item';
+export const EXPANDER_HEADER = 'expander-header';
+export const EXPANDER_ICON = 'expander-icon';
+// export const EXPANDER_CONTENT = 'expander-content';
+export const EXPANDER_CONTENT_TEXT = 'expander-content-text';
+
+export const ExpanderItem: Primitive<ExpanderItemProps, typeof Item> = ({
+  children,
+  className,
+  title,
+  ..._rest
+}) => {
+  const triggerId = useStableId();
+  const contentId = useStableId();
+  const { rest } = splitPrimitiveProps(_rest);
+  return (
+    <Item
+      className={classNames(ComponentClassNames.ExpanderItem, className)}
+      data-testid={EXPANDER_ITEM}
+      {...rest}
+    >
+      <Header
+        className={ComponentClassNames.ExpanderHeader}
+        data-testid={EXPANDER_HEADER}
```

Comment:
why do we need these test ids? only for testing purposes? I'd prefer to search by classname on tests rather than injecting an static value just to have a hook when running our tests suites. What do you think?

Reply:
We are using test ids in many places. I think it's a more stable API than searching by classnames. It draws a clear line between things used for styling (classnames) and things used for testing (data-testid). This allows both to be changed without side affects. It's also a best practice encouraged by React Testing Library.

Reply:
I get the benefits of using `test-ids`, but developer's HTML will contain unknown`test-id` attributes that might cause some confusion (at first). This is not a blocker, btw.